test/backend/routes.spec.js

```diff
@@ -319,10 +323,37 @@ describe('Authentication - ', () => {
                                  'https://bad-domain.plot.ly/v2/users/current failed, ' +
                                  'reason: getaddrinfo ENOTFOUND ' +
                                  'bad-domain.plot.ly bad-domain.plot.ly:443'));
-                    done();
-                })).catch(done);
+                    return;
```

Comment:
since this `setTimeout` isn't a promise, errors inside it won't get bubble out to the outermost promise and `.catch`. So,  this `.catch(done)` needs to stay